src/pivotal-ui/components/header/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "homepage": "http://styleguide.pivotal.io/objects.html#header",
   "dependencies": {
-    "@npmcorp/pui-css-drop-down-menu": "^1.3.4-0",
+    "@npmcorp/pui-css-drop-down-menu": "^1.3.4",
     "@npmcorp/pui-css-typography": "^4.0.0"
   },
-  "version": "3.0.0"
+  "version": "3.0.1-1"
```

Comment:
Ok, that was mainly for my own benefit. Hopefully we can start using tools to bump versions for us so it's no longer an issue.